data/sql/updates/pending_db_characters/rev_1638643807174948000.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_characters` (`sql_rev`) VALUES ('1638643807174948000');
+
+DROP TABLE IF EXISTS `character_settings`;
+CREATE TABLE IF NOT EXISTS `character_settings` (
+  `guid` INT UNSIGNED NOT NULL,
+  `source` TEXT NOT NULL,
+  `data` TEXT NULL,
+  PRIMARY KEY (`guid`, `source`)
```

Comment:
You should specify a length to `source` in order to make it primary key. Use varchar instead of text for that